experimental/dds/tree2/src/feature-libraries/chunked-forest/chunkedForest.ts

```diff
@@ -266,6 +267,14 @@ class ChunkedForest extends SimpleDependee implements IEditableForest {
 		);
 	}
 
+	public getDetachedFields(): DetachedField[] {
+		const fields: DetachedField[] = [];
+		for (const [key] of this.roots.fields) {
+			fields.push(keyAsDetachedField(key));
+		}
+		return fields;
```

Comment:
```suggestion
		return [...this.roots.fields.keys()]
```

Reply:
might need a type cast to make that actually work, but I still think its simpler.

Could also do 
```suggestion
		return [...this.roots.fields.keys()].map(keyAsDetachedField);
```

or probably best:
```suggestion
		return Array.from(his.roots.fields.keys(), keyAsDetachedField);
```